files/en/networking/tr069_acs/tr069_acs.md

```diff
@@ -293,3 +293,80 @@ Now we can see devices for the upgrade (from this group) and some parameters for
 Click on this button to update a device.
 
 ![ready for upgrade](ready_for_update.png)
+
+### Some features description
+
+<details>
+<summary><b>Relation with inventory</b></summary>
+<p markdown="1">
+It is possible  to assign device by ACS identifier (Serial number) to customer before device establishes first connection to ACS. In this way you can add an inventory item with a serial number specified and assign it to a customer:
+
+![item](item.png)
+
+After this you can setup some group and auto provisioning, for example to push PPPoE login&password, wifi SSID&password on first connect. After this with first connect a TR-069 device will be automatically assigned to a customer and initial configuration will be pushed:
```

Comment:
After this with first connect > During the first connection, ...;
and initial configuration > and an initial configuration